var/spack/repos/builtin/packages/ffb/package.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+
+
+class Ffb(Package):
+    """Computational Fluid Dynamics Software for aeroacoustics"""
+
+    homepage = "http://www.ciss.iis.u-tokyo.ac.jp/dl/index.php"
+    url      = "file://{0}/FrontFlow_blue.8.1.tar.gz".format(os.getcwd())
+    version('8.1', sha256='1ad008c909152b6c27668bafbad820da3e6ec3309c7e858ddb785f0a3d6e43ae')
+
+    patch('revocap_refiner.patch')
+
+    depends_on('mpi')
+    depends_on('blas')
+    depends_on('scalapack')
+
+    parallel = False
+
+    def install(self, spec, prefix):
+        d = find('.', 'make', recursive=True)
```

Comment:
It's probably better to hardcode the relative path from the stage root to the make directory.